_includes/footer.html

```diff
@@ -1,45 +1,60 @@
-<footer class="footer">
-  <ul>
-    <li class="footer__topic">
-      <h2 class="heading heading--capitalized">Info</h2>
-      <ul class="footer__topic-links">
-        <li>
-          <a href="#" class="link link--light">About HabitatMap</a>
-        </li>
-        <li>
-          <a href="#" class="link link--light">How It Works</a>
-        </li>
-        <li>
-          <a href="#" class="link link--light">FAQ</a>
-        </li>
-        <li>
-          <a href="#" class="link link--light">Donate</a>
-        </li>
-        <li>
-          <a href="#" class="link link--light">Get AirBeam</a>
+<footer class="footer panel">
+  <div class="split--half">
+    <ul class="footer__nav">
+      <li class="footer__topic">
+        <h2 class="footer__topic-heading heading heading--capitalized">Info</h2>
+        <ul class="footer__topic-links">
+          <li>
+            <a href="#" class="footer__link">About HabitatMap</a>
+          </li>
+          <li>
+            <a href="#" class="footer__link">How It Works</a>
+          </li>
+          <li>
+            <a href="#" class="footer__link">FAQ</a>
+          </li>
+          <li>
+            <a href="#" class="footer__link">Donate</a>
+          </li>
+          <li>
+            <a href="#" class="footer__link">Get AirBeam</a>
+          </li>
+        </ul>
+      </li>
+      <li class="footer__topic">
+        <h2 class="footer__topic-heading heading heading--capitalized">Connect</h2>
+        <ul class="footer__topic-links">
```

Comment:
and this `footer-column/group__list` or even `footer-links`, and then children could be `footer-links__element` like it's often in BEM

Reply:
or just `footer-link`